dev/scripts/generate-tarball.sh

```diff
@@ -33,14 +33,17 @@ mkdir -p ${WORK_DIR}
 HOME="${THIS}/../.."
```

Comment:
I actually think that Go is more portable than bash (running bash on Windows requires jumping through hoops)

portability aside, Go has additional benefits of being compiled and more importantly there seems to more consensus around what an API of a Go script is (command-line flags +  parameters) vs. shell script (environment variables, command-line arguments, parameters, and in some cases people assume it is okay to sed a bash script in which case every line of bash ends up being the API) ... the smaller the API surface, the easier it generally is to maintain and evolve the implementation

I could go on ... :-)